skimage/io/_io.py

```diff
@@ -130,6 +130,9 @@ def imsave(fname, arr, plugin=None, **plugin_args):
             plugin = 'tifffile'
     if is_low_contrast(arr):
         warn('%s is a low contrast image' % fname)
+    if (type(arr) == np.ndarray and arr.dtype == bool) or \
+        any(isinstance(ele, bool) for ele in arr):
```

Comment:
@soupault I made some small tests, and I can imagine this is used to deal with lists:
```
In : vector = [1, -1, True]
In : any(isinstance(elem, bool) for elem in vector)
Out: True
```
I am thinking like that because numpy binary elements have the `numpy.bool_` parameter:
```
In : nparr = np.array([True, False, True])
In : type(nparr[0])
Out: numpy.bool_
```
Thus, lists would not be checked by `(type(arr) == np.ndarray and arr.dtype == bool)`.